lib/shared/store.js

```diff
@@ -212,12 +213,18 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      if (_.isNil(action.data.speed)) {
+      if (_.isUndefined(action.data.speed)) {
         throw errors.createError({
           title: 'Missing state speed'
         })
       }
 
+      if (action.data.speed !== null && (!_.isNumber(action.data.speed) || !_.isFinite(action.data.speed))) {
```

Comment:
You can just do `!_.isNil(action.data.speed) || !_.isFinite(action.data.speed)` here, values that are not a number aren't finite either

Reply:
Oh, actually only isFinite() will suffice I realized